### ISSUE TITLE
Using numpy.stack instead of list zip

### ISSUE THREAD

**Issue by s1291** [marcomusy/vtkplotter#81]
In some examples (tunneling1, tunneling2, ...etc), the python list is used:

```
line = Line(list(zip(x, y))
```
This is two orders of magnitude slower than numpy.stack function

Here is a demonstration:

```python
import time
import numpy as np
import matplotlib.pyplot as plt

nvals = [10, 20, 40, 80, 160, 320, 640, 1280, 2560, 5120, 10240, 20480, 500000]
zip_times = []
stack_times = []

for n in nvals:
    x = np.linspace(0, 1, n)
    y = np.sin(x)

    # timing zip
    t0 = time.time()
    arr0 = np.array(list(zip(x, y)))
    t1 = time.time()
    zip_times.append(t1-t0)

    # timing stack
    t0 = time.time()
    arr1 = np.stack((x,y), axis=1)
    t1 = time.time()

    stack_times.append(t1-t0)

plt.semilogy(nvals, zip_times, label='zip')
plt.semilogy(nvals, stack_times, label='stack')
plt.xlabel('size N')
plt.ylabel('time in seconds')
plt.legend()
plt.show()

print(f"speedup (list zip time / np.stack time ): {np.array(zip_times) / np.array(stack_times)}")
```

![Figure_1](https://user-images.githubusercontent.com/17838489/70751781-7a427480-1d31-11ea-8e20-502f2a04094a.png)

**Comment by marcomusy** [comment]
Interestingly this gap reduces to factor a 10 when python lists are passed as arguments.
(e.g. `y = np.sin(x).tolist()`)
I'll replace the old fashioned list(zip()) to the numpy.stack() anywhere.
Thanks for drawing my attention to it! 

![image](https://user-images.githubusercontent.com/32848391/70800673-22e1ea00-1dad-11ea-8e84-281a990ee895.png)

**Comment by marcomusy** [comment]
fixed in the master along with #87
